third_party/move/move-compiler-v2/tests/checking/receiver/call_errors.move

```diff
@@ -24,6 +24,5 @@ module 0x42::m {
         s.receiver_ref_mut(1u8);
         s.receiver_ref_mut::<u8>(1);
         s.receiver_ref_mut::<u8, u8>(1);
-        s.receiver_needs_type_args(x)
```

Comment:
Add this to a separate test case because this error will be hidden by previous ones.

Reply:
This is caused by removing adhoc `finalize_type` in `post_process_receiver_call`.